### PR TITLE
sockstat: split FreeBSD, NetBSD and Linux implementations

### PR DESCRIPTION
**Pull request by vitorhcl**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

All options of all implementations are covered.

OpenBSD and macOS don't have `sockstat`.

### CODE REVIEW

File: pages/linux/sockstat.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# sockstat
+
+> List open Internet or UNIX domain sockets. Ported from FreeBSD.
+> Some of the displayed informations: USER, COMMAND, PID, FD (file descriptor), PROTO (protocol).
```
Comment (kbdharun) [inline pages/linux/sockstat.md:4]:
```suggestion
```
Is this line required?

File: pages/netbsd/sockstat.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# sockstat
+
+> List open Internet or UNIX domain sockets. Rewrite for NetBSD 3.0 from FreeBSD's `sockstat`.
+> Some of the displayed informations: USER, COMMAND, PID, FD (file descriptor), PROTO (protocol).
```
Comment (kbdharun) [inline pages/netbsd/sockstat.md:4]:
```suggestion
> List open Internet or UNIX domain sockets.
> Note: this program is a rewrite for NetBSD 3.0 from FreeBSD's `sockstat`.
```

File: pages/linux/sockstat.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# sockstat
+
+> List open Internet or UNIX domain sockets. Ported from FreeBSD.
```
Comment (kbdharun) [inline pages/linux/sockstat.md:3]:
```suggestion
> List open Internet or UNIX domain sockets.
```

File: pages/freebsd/sockstat.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# sockstat
+
+> List open Internet or UNIX domain sockets.
+> Some of the displayed informations: USER, COMMAND, PID, FD (file descriptor), PROTO (protocol).
```
Comment (kbdharun) [inline pages/freebsd/sockstat.md:4]:
```suggestion
```

File: pages/freebsd/sockstat.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+> More information: <https://man.freebsd.org/cgi/man.cgi?sockstat>.
+
+- View which users/processes are [l]istening to which ports:
```
Comment (gutjuri) [inline pages/freebsd/sockstat.md:7]:
```suggestion
- View which users/processes are [l]istening on which ports:
```

File: pages/freebsd/sockstat.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+`sockstat -l`
+
+- Show information for IPv[4]/IPv[6] sockets [l]istening on specific [p]orts using a specific [P]rotocol:
+
+`sockstat -{{4|6}} -l -P {{tcp|udp|sctp|divert}} -p {{port1,port2...}}`
+
+- Also show [c]onnected sockets, not resolving numeric UIDs to user [n]ames and using a [w]ider field size:
```
Comment (gutjuri) [inline pages/freebsd/sockstat.md:15]:
```suggestion
- Also show [c]onnected sockets, not resolving [n]umeric UIDs to user names and using a [w]ider field size:
```

File: pages/netbsd/sockstat.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+- Show information for IPv[4]/IPv[6] sockets [l]istening on specific [p]orts using a specific [P]rotocol:
+
+`sockstat -{{4|6}} -l -P {{tcp|udp|sctp|divert}} -p {{port1,port2...}}`
+
+- Also show [c]onnected sockets, showing [u]nix sockets:
+
+`sockstat -cu`
+
+- Only show [n]umeric output, not looking symbolic names for addresses and ports:
```
Comment (gutjuri) [inline pages/netbsd/sockstat.md:20]: wym by "not looking symbolic names for addresses and ports"?

**Review by gutjuri — COMMENTED** [review]
Thanks a lot for your contributions! I have some suggestions.

File: pages/netbsd/sockstat.md (hunk 7 above)
Comment (cyqsimon) [reply pages/netbsd/sockstat.md:20]:
```suggestion
- Only show [n]umeric output, without resolving symbolic names for addresses and ports:
```

Maybe this?

**Review by gutjuri — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**